### DISCUSSION TITLE
How stable is spiral linux

### DISCUSSION THREAD

**Discussion by Darahaas2001** [SpiralLinux/SpiralLinux-project#25] [General]
Hey there , just wondering how stable is spiral linux compared to other corporate backed distro like Ubuntu or Fedora

**Comment by geckolinux** [comment]
Hi there, thanks for the question. It all depends on your definition of "stable".

If we're talking about "stable" as in "not crashing", or "not buggy", they're probably all about the same, you'll win some and you'll lose some in each distro. Debian's philosophy is that new code introduces new bugs, even if that new code is intended to fix a known bug. So they basically don't fix bugs within the Stable release lifecycle unless it's extremely critical and/or security-related.

But if we define it as "not changing" or "tried and true", Debian is a bedrock in the Linux world. The whole reason why SpiralLinux was created was due to openSUSE's announcement that in a few years Leap will be going away or changing into a completely different sort of OS paradigm intended for corporate IT infrastructures. For desktop usage and as a base for GeckoLinux Static, that sort of OS model is simply not desirable. Fedora constantly debuts work-in-progress cutting-edge technology and is often the first to deprecate the older technology. Ubuntu is often criticized for starting and then abandoning projects that are sometimes poorly implemented or incomplete while they do exist. None of the above would happen in Debian, it's extremely conservative with implementing new stuff and even more cautious about throwing away old technology that is still relevant to even a small subset of its users.

**Reply by Darahaas2001** [reply]
Got it , soo the latest debian packages are fine to work with or shall I use gecko , i feel debian has got bigger repository and support compared to suse. Just that debian doesn't have latest nvidia drivers released 😅

**Reply by geckolinux** [reply]
Hi there, you'd really have to try them both for your needs. What I have observed is that with the official openSUSE repos plus the semi-official OBS repos (i.e. Graphics, Multimedia, ProAudio, etc.) plus selected personal OBS repos, there is no shortage of software available for openSUSE and it might even have more than Debian. But then again I always see posts on /r/openSUSE about major breakage with the NVIDIA drivers in Tumbleweed.
